.github/workflows/server-tests.yml

```
@@ -125,7 +125,7 @@ jobs:
 
     coverage:
         name: Coverage Wrap Up
-        if: secrets.CODECOV_TOKEN != ''
+        if: secrets.codecov_token != ''
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)